### PR TITLE
Updates the README with a troubleshoting and uninstallation section.

### PR DESCRIPTION
**Pull request by rmahroua**
@cigamit to follow up on the discussion, I updated the README to provide guidance around some troubleshooting items I have encountered while deploying the environment.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,41 +82,43 @@ Next we need to create user details.  First is the password set for all users.
...
+
+or if you are using separate var files for each workshop:
 ```
 ansible-playbook provision.yml -e @vars/myworkshop.yml
 ```
 
+It takes roughly one hour to provision an environment for a single student.  For multi student accounts, it is recommended you set the number of forks to the number of students.  The main reason is so the Tower installations to run all in parallel.
 
-It takes roughly 1 hour to provision an environment for a single student.  For multi student accounts, it is recommended you set the number of forks to the number of students.  The main reason is so the Tower installations to run all in parallel.
+All inventories will be placed in a folder named `workshop` in your local directory executing the playbook.  There are `student#-instances.txt` for each student there.  Also, there is an `instructor-inventory.txt`.  The student inventory is also placed on their tower host as `/etc/ansible/hosts`.
```
Comment (cigamit) [inline README.md]: the folder is workshops, with the s

File: README.md [hunk 2]
```diff
@@ -124,3 +129,51 @@ A student list / progress tracker is available for admins at
...
 ```
 You are required to enter your Domain Admin password to login to see the list.
+
+## Deleting the environment
+
+Run the `teardown.yml` playbook to tear down the environment. The playbook deletes the instances, the VPCs, and any other resource that the installer creates.
+It takes roughly five minutes for this playbook to run.
+
+```
+ansible-playbook teardown.yml
```
Comment (cigamit) [inline README.md]: We might want to make a note that if you have multiple workshops running at the same time, then you should point the vars file for the workshop you want to teardown.  Such as 
ansible-playbook teardown -e @vars/AnsibleAutomatesDallas.yml
I have had up to 3 workshops provisioned at once, so its a good habit to get into.

**Comment by rmahroua** [comment]
@cigamit I updated the file per your request. I also revisited a bit the explanation of what the workshops directory contains. Let me know what you think.

File: README.md [hunk 3]
```diff
@@ -79,41 +82,53 @@ Next we need to create user details.  First is the password set for all users.
...
 ansible-playbook provision.yml
 ```
-or if you are using separate var files for each workshop
+
+or if you are using separate var files for each workshop:
 ```
 ansible-playbook provision.yml -e @vars/myworkshop.yml
 ```
 
+It takes roughly one hour to provision an environment for a single student.  For multi student accounts, it is recommended you set the number of forks (in `ansible.cfg`) to the number of students.  The main reason is that it instructs Ansible to run all of theTower installations in parallel.
```
Comment (cigamit) [inline README.md]: missed a space in theTower

**Pull request merged**